ui/src/components/helpers/SermonNotes.js

```diff
@@ -19,6 +19,35 @@ export const getBiblePassage = async (passage, currentText) => {
     }
     const [chapter, verse] = chapterAndVerse.split(':');
 
+    // handle if its different chapter
+    const isDifferentChapter = chapterAndVerse.split(':').length > 2;
+    if (bible && bible[book] && isDifferentChapter) {
+      const [firstChapterVerse, lastChapterVerse] = chapterAndVerse.split('-');
+      const [firstChapter, firstVerse] = firstChapterVerse.split(':');
+      const [lastChapter, lastVerse] = lastChapterVerse.split(':');
```

Comment:
So it will work max 2 chapters right?

Reply:
Nope, it shod work with any length of chapters